content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,183 +2,209 @@
+calculations.
+
+.. image:: sla/sla-pivot-measures.png
+   :align: center
+   :alt: View of the available measures in the SLA status analysis report.
 
-To add the number of SLAs passed or in progress, click the :guilabel:`Measures` button to reveal a
-drop-down menu of reporting criteria, and choose from the options available based on the
-measurements preferred. Whenever a measurement is picked, a check mark will appear in the drop-down
-menu to indicate that that measurement is included, and a corresponding new column will emerge in
-the pivot table to show the relevant calculations.
+To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to the policy name and
+then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon next to the
+policy name.
 
-To add a group to a row or column, click the plus :guilabel:` + ` button next to
-:guilabel:`Total`, and then select one of the groups. To remove one, click the minus :guilabel:` - `
-button and deselect.
+.. image:: sla/sla-pivot-groups.png
+   :align: center
+   :alt: View of the available group by options in the SLA status analysis report.
 
 Using the Graph view
 --------------------
 
-The :guilabel:`Status Analysis` report can also be viewed as a :guilabel:`Bar`, :guilabel:`Line`, or
-:guilabel:`Pie` Chart. Toggle between these views by selecting the appropriate icon at the top of
-the chart.
+The :guilabel:`SLA Status Analysis` report can also be viewed as a :guilabel:`Bar`,
+:guilabel:`Line`, or :guilabel:`Pie` Chart. Toggle between these views by first selecting the
+:guilabel:`Graph` icon at the top-right of the dashboard. Then select the appropriate chart icon at
+the top-left of the graph.
 
 .. tabs::
 
    .. tab:: Bar Chart
 
        .. image:: sla/sla-report-bar.png
           :align: center
-          :alt: View of the SLA status analysis report in bar view
+          :alt: View of the SLA status analysis report in bar view.
 
    .. tab:: Line Chart
 
        .. image:: sla/sla-report-line.png
           :align: center
-          :alt: View of the SLA status analysis report in line view
+          :alt: View of the SLA status analysis report in line view.
 
    .. tab:: Pie Chart
 
        .. image:: sla/sla-report-pie.png
           :align: center
-          :alt: View of the SLA status analysis report in pie chart view
+          :alt: View of the SLA status analysis report in pie chart view.
 
 .. tip::
-   Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` can be viewed :guilabel:`Stacked`.
-   This presents two or more groups to appear on top of each other instead of next to each other,
-   making it easier to compare data.
+   Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` can be viewed :guilabel:`Stacked` by
+   selecting the :guilabel:`Stacked` icon. This presents two or more groups to appear on top of each
+   other instead of next to each other, making it easier to compare data.
```

Comment:
```suggestion
   Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` views can be :guilabel:`Stacked` by
   selecting the :guilabel:`Stacked` icon. This displays two or more groups on top of each other,
   instead of next to each other, making it easier to compare data.
```